indico/modules/events/registration/controllers/management/tickets.py

```diff
@@ -90,7 +102,14 @@ def _checkProtection(self):
 
     def _process(self):
         filename = secure_filename('{}-Ticket.pdf'.format(self.event_new.title), 'ticket.pdf')
-        return send_file(filename, generate_ticket(self.registration), 'application/pdf')
+        template = self.regform.ticket_template
+        if template is None:
+            template = get_default_template_on_category(self.event_new.category)
```

Comment:
why not simply

    template = self.regform.ticket_template or get_default_template_on_category(self.event_new.category)